modules/fbs-core/web/src/app/tool-components/math-input/math-input.component.ts

```diff
@@ -62,5 +62,10 @@ export class MathInputComponent implements OnChanges, AfterViewInit {
         el.expression = JSON.parse(this.defaultValue.mathJson);
       }
     }
+    if (this.disabled) {
+      var style = document.createElement("style");
+      style.innerHTML = ".ML__content { overflow: unset; }";
+      this.input.nativeElement.shadowRoot.appendChild(style);
```

Comment:
```suggestion
      var sheet = new CSSStyleSheet();
      sheet.insertRule(".ML__content { overflow: unset; }");
      this.input.nativeElement.shadowRoot.adoptedStyleSheets = [sheet];
```


I would use [CSSStyleSheet](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet) for this, because then you don't have to create a style element.